org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/IOUtils.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4xml.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * IO utilities class
+ * 
+ * @author Angelo ZERR
+ *
+ */
+public class IOUtils {
+
+	/**
+	 * Convert the given input stream in a String.
+	 * 
+	 * @param is the input stream
+	 * @return the given input stream in a String.
+	 */
+	public static String convertStreamToString(InputStream is) {

```

Comment:
Scanner is autocloseable and its close() method automatically closes the source, so you can simplify this to:
```java
	public static String convertStreamToString(InputStream is) {
		try (Scanner s = new java.util.Scanner(is)) {
			s.useDelimiter("\\A");
			return s.hasNext() ? s.next() : "";
		}
	}
```

Reply:
@fbricon updated